intelligence/tasks.py

```
@@ -92,4 +93,149 @@
+        from gitdm.models import PatientProfile
+        
+        # دریافت بیماران فعال (که در 3 ماه گذشته داده داشته‌اند)
+        cutoff_date = timezone.now() - timezone.timedelta(days=90)
+        
+        active_patients = PatientProfile.objects.filter(
+            Q(labresult__taken_at__gte=cutoff_date) |
+            Q(encounter__occurred_at__gte=cutoff_date)
+        ).distinct()
+        
+        total_processed = 0
+        total_alerts = 0
```

Ruff (0.12.2)

Local variable `total_alerts` is assigned to but never used

Remove assignment to unused variable `total_alerts`

(F841)